inventory/group_vars/k8s-cluster.yml

```diff
@@ -132,3 +132,8 @@ efk_enabled: false
 
 # Helm deployment
 helm_enabled: false
+
+# dnsmasq
+dnsmask_upstream_dns_servers:
+  - /resolvethiszone.with/10.0.4.250
+  - 8.8.8.8
```

Comment:
Why is this here? We already have a system_and_upstream_dns_servers section for dnsmasq config.